src/collections/_documentation/workflow/alerts-notifications/index.md

```diff
@@ -20,15 +22,25 @@ Metric Alerts are currently in beta. For more information about access to Metric
     level="warning"
 %}
 
-Metric alerts allow you to filter and set thresholds on all errors across a project. Use metric alerts for high-level monitoring of patterns or fine-grained monitoring of individual events. Metric alerts help express lightweight service-level objectives (SLOs) such as users affected by signup-page errors or volume of database errors.
+Use Metric Alerts to:
+- Alert on service-level metrics. Is my service affecting more than X users?
+- Alert on events across issues. How many errors on the signup page?
+- Alert when a metric goes below a threshold. For example, when the number of active users has drastically dropped.
+- Create alerts more quickly and intuitively. A visual interface that allows you to explore and refine.
+
+For more details, see the [full documentation on Metric Alerts]({%- link _documentation/workflow/alerts-notifications/alerts.md -%}#metric-alerts).
 
 ### Issue Alerts
 
-Issue alerts fire whenever **any issue** in the project matches the specified criteria, such as a resolved issue re-appearing or an issue affecting many users.
+Use Issue Alerts to:
+- Alert when an individual issue is problematic. Any new bug on the checkout page? Any single issue affecting more than X users?
+- Alert on issue-specific concepts. For example, new issues and regressed issues.
+- Combine multiple conditions with the `OR` and `NONE` operators.
+- Alert based on Issue Owner rules.
 
-For more details, see the [full documentation on Issue Alerts]({%- link _documentation/workflow/alerts-notifications/alerts.md -%}#issue-alerts).
+Some alert settings apply at the project level and to all alerts in the project. Other alert settings are specific to the alert. You can tweak your [personal alerts settings]({%- link _documentation/workflow/alerts-notifications/alerts.md -%}#alert-subscription) to control which alerts you receive.
```

Comment:
Swap this paragraph with the one below?

Reply:
Typically, Docs puts the "For more details..." links at the end of a section. I updated it to indicate what the "details" will be about.

Reply:
i thought the settings comment applied generally to alerts, not to issue alerts specifically?

Reply:
Yep. I didn't update line 41. I updated line 43. Line 43 has the link with more details.

Reply:
I assumed you wanted to swap line 41 with line 43 (the line below). So I just changed line 43 to keep it consistent with the rest of Docs.